lib/kramdown/parser/gfm.rb

```diff
@@ -60,6 +64,30 @@ def add_hard_line_breaks(element)
       FENCED_CODEBLOCK_MATCH = /^(([~`]){3,})\s*?(\w[\w-]*)?\s*?\n(.*?)^\1\2*\s*?\n/m
       define_parser(:codeblock_fenced_gfm, /^[~`]{3,}/, nil, 'parse_codeblock_fenced')
 
+      STRIKETHROUGH_DELIM = /~~/
+      STRIKETHROUGH_MATCH = /#{STRIKETHROUGH_DELIM}[^\s~](.*)[^\s~]#{STRIKETHROUGH_DELIM}/m
+      define_parser(:strikethrough_gfm, STRIKETHROUGH_MATCH, '~~')
```

Comment:
Why are you potentially matching more than needed? It should be sufficient to use `STRIKETHROUGH_DELIM` here instead of `STRIKETHROUGH_MATCH`.